content/components/nav-list.mdx

```diff
@@ -0,0 +1,143 @@
+      width="456"
+      src="https://user-images.githubusercontent.com/2313998/226925595-ab08263b-600d-43be-b56a-c9b6af8950a1.png"
+    />
+    <Caption>Use leading visuals consistently.</Caption>
+  </Do>
+  <Dont>
+    <img
+      role="presentation"
+      width="456"
+      src="https://user-images.githubusercontent.com/2313998/226925597-717c5e21-5792-4d4c-94bc-5a798ca486ce.png"
+    />
+    <Caption>Avoid mixing items that have leading visuals with items that don't.</Caption>
+  </Dont>
+</DoDontContainer>
+
+<DoDontContainer>
+  <Do>
+    <img
+      role="presentation"
+      width="456"
+      src="https://user-images.githubusercontent.com/2313998/226925601-ad39870d-56d4-4696-911b-5f05ccd436d4.png"
+    />
+    <Caption>Only use dividers to visually group items.</Caption>
+  </Do>
+  <Dont>
+    <img
+      role="presentation"
+      width="456"
+      src="https://user-images.githubusercontent.com/2313998/226925603-d17f11b7-1c80-42d8-bd0e-4fc329ac6e2b.png"
+    />
+    <Caption>Don't use dividers to separate individual items.</Caption>
+  </Dont>
+</DoDontContainer>
+
+## Anatomy
+
+Nav list items are rendered using [action list](/components/action-list) items, so they share anatomy, styles, and some options.
+
+## Options
+
+### Leading and trailing visuals
+
+<img
+  width="960"
+  role="presentation"
+  src="https://user-images.githubusercontent.com/2313998/226925606-c1d6d354-5692-410e-a27e-4e25f1c264ca.png"
+/>
+
+**Leading visuals** are commonly used to show a visual hint for what the item is. For example, an icon or avatar that represents the view being linked to.
+
+**Trailing visuals** are commonly used to show auxiliary information about the item. For example, a count of unread notifications on the page.
+
+Nav list items may have both a leading and a trailing visual.
+
+### Grouped list sections
+
+<img
+  width="960"
+  role="presentation"
+  src="https://user-images.githubusercontent.com/2313998/226925613-71c86520-394d-46b7-a29d-090268d4e0cb.png"
+/>
+
+You can semantically group your nav list into multiple lists using groups. You can choose whether or not to show a label for each group.
+
+If you want to visually group your nav list into sections but don't want the semantics of nested `<ul>`s, you may use dividers to visually break your list into sections.
+
+Grouping should be used to make long or complex nav lists easier to comprehend.
+
+### Sub-items
+
+<img
+  width="960"
+  role="presentation"
+  src="https://user-images.githubusercontent.com/2313998/226925617-c5a27545-905f-491e-9059-6389264ab77d.png"
+/>
+
+Sub-items may be used to render nested navigation structures. Only a single level of nesting is supported. If you need to handle a deeply-nested structure, consider using a [tree view](/components/tree-view)
+
+Sub-items may be collapsed or expanded under a top-level nav list item.
+
+The top-level nav list item cannot be a link; it only behaves as a button that expands and collapses the sub-items.
```

Comment:
`behaves as a toggle` feels clearer than `behaves as a button`, but i do not feel strongly about this. also dont want to confuse with our toggle component, so maybe this is fine